factory/processing/processComponents_test.go

```diff
@@ -1012,6 +1023,8 @@ func getRunTypeComponents(rt runType.RunTypeComponentsHolder) *mainFactoryMocks.
 		GenesisMetaBlockChecker:             rt.GenesisMetaBlockCheckerCreator(),
 		NodesSetupCheckerFactoryField:       rt.NodesSetupCheckerFactory(),
 		EpochStartTriggerFactoryField:       rt.EpochStartTriggerFactory(),
+		LatestDataProviderFactoryField:      rt.LatestDataProviderFactory(),
```

Comment:
:+1: 
Can we also have a test for nil `LatestDataProviderFactoryField`? We missed this in the previous PR

Reply:
Test was added in previous PR